sql/catalyst/src/main/scala/org/apache/spark/sql/AnalysisException.scala

```diff
@@ -54,3 +54,13 @@ class AnalysisException protected[sql] (
     s"$message;$lineAnnotation$positionAnnotation"
   }
 }
+
+/**
+ * Thrown when a query failed for invalid function argument size, usually because query pass
```

Comment:
`usually because a SQL function is called with the wrong number of arguments.`

Reply:
> usually because a SQL function is called with the wrong number of arguments.

Done